skopt/searchcv.py

```diff
@@ -398,6 +399,146 @@ def add_spaces(self, names, search_spaces):
         for space, name in zip(search_spaces, names):
             self.search_spaces_[name] = space
 
+    # copied for compatibility with 0.19 sklearn from 0.18 BaseSearchCV
+    @property
+    def best_score_(self):
```

Comment:
We don't use `best_score_` and `best_params_` for our code right? So I think we should not retrofit them, that is somehow between the user and sklearn I think. My motivation is to keep `BayesSearchCV` as similar as possible in interface to what ever version of sklearn the user has.